config/application.rb

```diff
@@ -22,8 +22,14 @@ class Application < Rails::Application
 
     # Do not swallow errors in after_commit/after_rollback callbacks.
     config.active_record.raise_in_transactional_callbacks = true
-    
+
     # Gzip files when possible
     config.middleware.use Rack::Deflater
+
+    #HCaseflow application urls
+    config.app_urls = {"Caseflow" => "https://caseflow.ds.va.gov", "Efolder" => "https://efolder.cf.ds.va.gov"}
```

Comment:
I was going for a shorter URL and it's less error prone when it comes to how the app parameter name should look like.